graviti/manager/sheets.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""The implementation of the Sheets."""
+
+from typing import Any, Dict, Iterator, MutableMapping
+
+from tensorbay.dataset import Notes, RemoteData
+from tensorbay.label import Catalog
+from tensorbay.utility import URL
+
+from graviti.client import get_catalog, get_notes, list_data_details, list_segments
+from graviti.dataframe import DataFrame
+from graviti.portex import Extractors, catalog_to_schema, get_extractors
+from graviti.utility import LazyFactory, LazyList, NestedDict
+
+LazyLists = NestedDict[str, LazyList[Any]]
+
+
+class Sheets(MutableMapping[str, DataFrame]):
+    """The basic structure of the Graviti sheets."""
+
+    _data: Dict[str, DataFrame]
+    _dataset_id: str
+    access_key: str
+    url: str
+    commit_id: str
+
```

Comment:
do we need a `__init__` method to init these members?

Reply:
We will do that after the